### PR TITLE
profile.md and tcpconnect.md : final typos from full doc reading

### PR DESCRIPTION
**Pull request by didier-durand**
Hi there,

The final set of typos that I found after reading full doc.

Your product is very interesting: keep up!

Didier

Signed-off-by: Didier Durand <durand.didier@gmail.com>

### CODE REVIEW

File: docs/guides/profile.md [hunk 1]
```diff
@@ -13,8 +13,8 @@ pod/random created
 ```
 
 Using the profile gadget, we can see the list of stack traces.
-The following command filter only for pods named "random", execute the command
-and interrupt it after ~30 seconds. The `-K` option is passed to show only the
+The following command filters only for pods named "random", executes the command
```
Comment (mauriciovasquezbernal) [inline docs/guides/profile.md:16]: I think "execute" and "interrupt" are correct as they're imperative mode.

**Review by mauriciovasquezbernal — COMMENTED** [review]
Thanks a lot for the PR. Just a comment about some of the changes.

**Comment by didier-durand** [comment]
@mauriciovasquezbernal : with your explanation, I understand now how I should have read this text. Now clear, so I restored the original imperative for those 2 verbs.  See latest commit.Thanks! Didier

**Review by alban — APPROVED** [review]
Thanks

**Pull request merged**